store/migration/dev/20221027000000##approval_instance.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE approval_instance ( 
```

Comment:
Missing comment. Also "approval_instance stores plugin application approval instances." is not very clear.

Reply:
Changed to external_approval ?